### ISSUE TITLE
Chapter 3 - The Sudden Appearance of player-idle-east

### ISSUE THREAD

**Issue by mprogers** [Apress/Devel-2D-Games-Unity#20]
In figure 3-40, the preview source state is player-idle-east, but AFAICT it's never been mentioned previously.

**Comment by titoolazabal** [comment]
right.  And in page 89, second paragraph from the top, it says "..adjust each idle animation to 0.25."  I thought we had only one, player-idle.

**Comment by JaredHalpern** [comment]
Thank you so much for mentioning. I'll take a look and update the errata as needed.

**Comment by jwinters733** [comment]
Is there any update on this? I'm curious on how to make the player idle in a certain direction once we have stopped walking in that direction
